06-building-a-program-start-to-finish/solving-the-problem/output/invertedindexutil.js

```diff
@@ -0,0 +1,276 @@
+/* eslint-disable no-unused-vars */
+/* eslint-disable no-undef */
+
+/**
+ * @class InvertedIndexUtility
+ */
+class InvertedIndexUtility {
+/**
+   * filterIndexed method takes an already indexed array
+   * and filters it to match the query
+   * @param {Array} query - An array of the words
+   * gotten from the sanitized query
+   * @param {Array} indexedFile - An already indexed file
+   * @return {Array} - The queried indexed array
+   * @memberOf InvertedIndexUtility
+   */
+  static filterIndexed(query, indexedFile) {
+    const result = [];
+    query.forEach((term) => {
+      indexedFile.forEach((index) => {
+        if (term === index[0]) {
+          result.push(index);
+        }
+      });
+    });
+    return result;
+  }
+
+  /**
+   * sanitizeQuery method removes unwanted terms in the query string
+   * and also converts the query string into an array of each unique term
+   * @static
+   * @param {String} query - The terms to search for the index
+   * @returns {Array} - The already formatted query
+   * @memberOf InvertedIndexUtility
+   */
+  static sanitizeQuery(query) {
```

Comment:
Your method and parameter names are inappropriate. Please names that best describes them.